console/calling_commands.rst

```diff
@@ -8,26 +8,32 @@ or if you want to create a "meta" command that runs a bunch of other commands
 changed on the production servers: clearing the cache, generating Doctrine
 proxies, dumping web assets, ...).
```

Comment:
```suggestion
proxies, dumping web assets, ...)::
```

and remove the following code block

Reply:
I do not agree with this. This paragraph is not related to the below code block. The `::`, which is rendered as `:`, marks a relation between the paragraph before it and the block after it (whether it be a list or a code block).

Reply:
Hmm not sure but we use it across the whole docs like this. Maybe there are other occurrences we didn't check?

How shall we proceed here? Add it to the whitelist?

cc @javiereguiluz

Reply:
I'm not 100% sure about this. In some edge-cases we use `.. code-block:: php` (or `even just `::`) at the beginning of some line to show a code example.

But most of the times there's a relevant paragraph before the code sample, so the `::` is natural.

In this case the first paragraph is not directly related, so it's better to not add the `::`

But then, if we don't add it, the code sample comes a bit "abrupt".

We could solve it by adding a short introductory phrase to the code (like it was before) ... but that phrase is a bit "silly"/empty ... so why add it?

So, I'd vote to do the changes suggested by Wouter (but this is certainly tricky 😓 )

Reply:
Btw, I removed the original line because `[...] is straightforward` for me feels the same as "[...] is simple/easy", which is something we actively remove the from the docs the past years. I'm happy to add another line, but as Javier said it felt "silly".

I think the rule should be "use `::` if the previous sentence ended with an `:` already and the current line is `.. code-block:: php`".

Reply:
Hmm we can adjust this rule but it's more complex than needed IMO.

If it does not belong to the paragraph before I would argue that "starting" with a code block is not a soft introduction. What about adding a sub headline and/or an introduction? 

Don't get me wrong, it's ok to adjust this rule but it would be the first and only case for now 🤔

Reply:
Alright, changed order of code block/description.

However, I do think that this rule is too strict to enforce everywhere. Both orders have the same readability for me and adding text just to introduce a code example only increases read time imho.

Reply:
Can you please open an issue on DOCtor-RST repo to keep that in mind? Thanks!